ios/Podfile

```diff
@@ -41,9 +41,9 @@ def shared_pods
   pod 'RNReanimated', :path => '../node_modules/react-native-reanimated'
 
   permissions_path = '../node_modules/react-native-permissions/ios'
-  pod 'Permission-BluetoothPeripheral', :path => "#{permissions_path}/BluetoothPeripheral.podspec"
-  pod 'Permission-Notifications', :path => "#{permissions_path}/Notifications.podspec"
+  pod 'Permission-LocationWhenInUse', :path => "#{permissions_path}/LocationWhenInUse.podspec"
   pod 'Permission-LocationAlways', :path => "#{permissions_path}/LocationAlways.podspec"
+  pod 'Permission-Notifications', :path => "#{permissions_path}/Notifications.podspec"
```

Comment:
not related to this PR, but this pod should be moved over to the `gps` target, and any residual permissions for location or bluetooth should be removed from the `BTE` `info.plist`

Reply:
yup, we'll need to tackle this in a future commit.